### DISCUSSION TITLE
Add remix-run/serve dependency for Vanilla exported projects

### DISCUSSION THREAD

**Discussion by Barabasbalazs** [webstudio-is/webstudio-community#196] [Ideas]
When generating a project build from the builder app it would be nice to include the [remix-run/serve](https://remix.run/docs/en/main/other-api/serve) dependency with a start command.
This way if the user can immediately serve the created production build. At the moment the package.json provides scripts for running a dev server and creating a production build, but none for running the production build as well.

![Screenshot_20241012_195131](https://github.com/user-attachments/assets/2196d64e-6cc0-409b-8b3b-960d6c6ebfb8)

I was thinking something along these lines: 

![package-json=ss](https://github.com/user-attachments/assets/a4ee5872-a660-413b-a247-349377bcab4a)

**Comment by kof** [comment]
`npx serve .`  doesn't work for you?

**Reply by Barabasbalazs** [reply]
It does work, but that's just a static file server. If I'd like to use the Remix app that gets built by running the build command, I need to have some kind of server behind it.
```bash
npx serve .
```
is only able to serve the files only, it can't use the functionality that comes with a Remix app that I receive in the _build_ folder

![wsstudio_ss](https://github.com/user-attachments/assets/416848e4-0bb8-4645-89d1-77a1e0f25a3e)
